Library/Homebrew/cmd/audit.rb

```diff
@@ -411,7 +411,7 @@ def audit_desc
       problem "Description shouldn't start with an indefinite article (#{$1})"
     end
 
-    if desc =~ /^#{formula.name}\s/i
+    if desc =~ /^#{Regexp.escape formula.name}\s/i
```

Comment:
Can we use `start_with?` directly?  e.g. `desc.downcase.start_with? "#{formula.name} "`

The main reason is regex has poor performance compared with direct string matching.